CI_JobHistory.py

```diff
@@ -174,6 +203,19 @@ def main():
                     spy_links = monitor.get_jobs_with_date(ci_link,start_date,end_date)
                     get_failed_testcases(spy_links,zone=args.zone)
                     monitor.final_job_list = []
+            
+            if option == '5':
+                tc_name = input("Enter the testcase names comma seperated: ")
+                tc_list =  tc_name.split(",")
+
+                for ci_name,ci_link in ci_list.items():
+                    print("-------------------------------------------------------------------------------------------------")
+                    print(ci_name)
+                    spy_links = monitor.get_jobs_with_date(ci_link,start_date,end_date)
+                    for tc in tc_list:
+                        print("TESTCASE NAME: " + tc)
+                        get_testcase_failure(spy_links,zone=args.zone,tc_name=tc)
 
```

Comment:
Please make monitor.final_job_list = [] as empty because monitor.final_job_list variable captures all builds run in a CI if it is not made empty in second iteration it will query again for the jobs run in first CI.

Reply:
Done.